<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjPadClass</class>
 <widget class="QMainWindow" name="ProjPadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjPad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuMarkdown">
    <property name="title">
     <string>Markdown</string>
    </property>
    <widget class="QMenu" name="menuHeader">
     <property name="title">
      <string>Heading</string>
     </property>
     <addaction name="actionHeading_1"/>
     <addaction name="actionHeading_2"/>
     <addaction name="actionHeading_3"/>
     <addaction name="actionHeading_4"/>
     <addaction name="actionHeading_5"/>
     <addaction name="actionHeading_6"/>
    </widget>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="menuHeader"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMarkdown"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>Bold (**...**)</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>Italic (*...*)</string>
   </property>
  </action>
  <action name="actionHeading_1">
   <property name="text">
    <string>Heading 1 (# ...)</string>
   </property>
  </action>
  <action name="actionHeading_2">
   <property name="text">
    <string>Heading 2 (##...)</string>
   </property>
  </action>
  <action name="actionHeading_3">
   <property name="text">
    <string>Heading 3 (###...)</string>
   </property>
  </action>
  <action name="actionHeading_4">
   <property name="text">
    <string>Heading 4 (####...)</string>
   </property>
  </action>
  <action name="actionHeading_5">
   <property name="text">
    <string>Heading 5 (#####...)</string>
   </property>
  </action>
  <action name="actionHeading_6">
   <property name="text">
    <string>Heading 6 (######...)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProjPad.qrc"/>
 </resources>
 <connections/>
</ui>
